lib/generate-suggestions.ts

```diff
@@ -43,11 +43,14 @@ async function generateSuggestions(flags: {
   console.log(JSON.stringify(results, null, 2));
 }
 
-async function findComponentsInJSXFiles(params: {
-  directory: string;
-  files?: string[];
-  components: FetchComponentResponse;
-}): Promise<{ [apiId: string]: Result }> {
+async function findComponentsInJSXFiles(
+  params: (
+    | { directory: string; files?: string[] }
+    | { files: string[]; directory?: string }
```

Comment:
I fixed this type definition to be correct -- only one of `directory` or `files` is required according to the existing implementation